source/tunneling/SecureTunnelingContext.h

```diff
@@ -46,26 +46,31 @@ namespace Aws
                         uint16_t port,
                         const OnConnectionShutdownFn &onConnectionShutdown);
 
+                    /**
+                     * \brief Constructor
+                     */
+                    SecureTunnelingContext() = default;
+
                     /**
                      * \brief Destructor
                      */
-                    ~SecureTunnelingContext();
+                    virtual ~SecureTunnelingContext();
```

Comment:
Do we need to explicitly declare the virtual function since it's not being implemented inline anyways?